docs/CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ nav_order: 5
 
     *Yoshiyuki Hirano*
 
+* Update link to benchmark script.
```

Comment:
```suggestion
* Update link to benchmark script in docs.
```

Reply:
Thanks for the change! I guess I was a little too brief.